moninag/static/src/ts/app/navs/side-nav/side-nav.component.ts

```diff
@@ -33,6 +77,14 @@ export class SideNavComponent implements OnInit {
             })    
     }
 
+    onSelect(server:Server): void {
+        this.selectedServer = server;
+    }
+
+    onSelect2(service:Service): void {
```

Comment:
Rename this method, bad practices used a number for the names.
for example: onSelectedServer, onSelectedService